UserManagement/src/main/resources/application.properties

```diff
@@ -0,0 +1,4 @@
+#Local MongoDB config
```

Comment:
Create a technical story to externalize the application .properties in the next milestone.

Reply:
Will look for different options for externalizing and move ahead accordingly